alacritty_terminal/src/tty/windows/child.rs

```diff
@@ -85,6 +90,14 @@ impl ChildExitWatcher {
     pub fn deregister(&self) {
         *self.interest.lock().unwrap() = None;
     }
+
+    pub fn fd(&self) -> isize {
+        self.child_handle
+    }
```

Comment:
This is generally not a `FD`? You should also document what exactly is getting returned here.

Reply:
Sorry for the inconvenience, I'm still learning, not only coding but also how to collaborate. English is not my native language, so sometimes my ideas may not be expressed very accurately, please understand. Here, the return is the handle of the process. In the Windows environment, a handle is represented as an `isize`. Returning isize really confusing here, I will write as the `HANDLE` type. This handle can mainly be used to obtain the corresponding process ID (`pid`) through the Windows API `GetProcessID`.
I will convert this pr to draft.

Reply:
No, I mean, if you name it as `fn handle` and not `fn fd` it'll be better, and also if you add a `///` explaining you're actually returning, for example with a link to microsoft docs for that.

Also, it should be for the type visible for users, so they can get a grasp of what it is on docsrs.